linux/build.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
+set -eu
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+. "$(dirname "$THIS_SCRIPT")/../resources/build/build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+################################ Main script ################################
+
+# TODO: add keyman-config as child project
+
+cd "$THIS_SCRIPT_PATH"
+
+OUTPUT_PATH=
+
+builder_describe \
+  "Build Keyman for Linux." \
+  "clean" \
+  "configure" \
+  "build+" \
+  "test" \
+  ":engine                   ibus-keyman" \
+  "all                       clean, configure, and build" \
+  "install                   install artifacts" \
+  "uninstall                 uninstall artifacts" \
+  "--debug,-d                Debug build" \
+  "--output=OUTPUT_PATH,-o   Output path (default: build/)"
+
+builder_parse "$@"
+
+if builder_has_action all; then
+  _builder_chosen_action_targets=("clean:engine" "configure:engine" "build:engine")
+fi
```

Comment:
I'm not comfortable with this, because it's touching internal variables for builder, which are subject to change, and maintenance on builder should not require auditing usage of its internal variables in other scripts.

Reply:
ok, I'll remove `all`, then we don't need this. This can be mostly done with internal dependencies.

However, it seems there's still a problem with internal dependencies, or I still don't understand how things work: With the outputs defined, when I run `./build.sh --debug build` it'll always run the `configure` action, even though `configure` ran before:

```
Automatically running following required actions with missing outputs:
* configure:engine
## [linux/ibus-keyman] configure starting...
Directory already configured.

Just run your build command (e.g. ninja) and Meson will regenerate as necessary.
If ninja fails, run "ninja reconfigure" or "meson --reconfigure"
to force Meson to regenerate.

If build failures persist, run "meson setup --wipe" to rebuild from scratch
using the same options as passed when configuring the build.
To change option values, run "meson configure" instead.
## [linux/ibus-keyman] configure completed successfully
## [linux/ibus-keyman] build starting...
ninja: no work to do.
## [linux/ibus-keyman] build completed successfully
```

Reply:
My comment was in the wrong file. I changed `linux/ibus-keyman/build.sh` which does define outputs.